sql/core/src/test/resources/sql-tests/results/subquery/in-subquery/in-basic.sql.out

```diff
@@ -41,15 +41,15 @@ select 1 from tab_a where (a1, b1) not in (select (a2, b2) from tab_b)
 struct<>
 -- !query 4 output
 org.apache.spark.sql.AnalysisException
-cannot resolve '(named_struct('a1', tab_a.`a1`, 'b1', tab_a.`b1`) IN (listquery()))' due to data type mismatch: 
-The number of columns in the left hand side of an IN subquery does not match the
+cannot resolve 'named_struct('a1', tab_a.`a1`, 'b1', tab_a.`b1`) IN (listquery())' due to data type mismatch: 
+The number of columns in the left hand side of a predicate subquery does not match the
```

Comment:
@francis0407 Depending upon the answer to my previous question, we may not need a change here.